src/HPolygon.jl

```diff
@@ -10,7 +10,8 @@ are sorted in counter-clockwise fashion with respect to their normal directions.
 
 ### Fields
 
-- `constraints`       -- list of linear constraints, sorted by the angle
+- `constraints`       -- list of linear constraints, sorted by the angle in
```

Comment:
this was there before, `.. sorted by the angle` -- is this correct? if the algorithm may start in any position, then the constraints will not be sorted by the angle (by "the angle" i understand `a1 < a2 < ... < an` where `ai` is the angle)

Reply:
True, I think we agreed that we want to give up this guarantee. Note that our sorting mechanisms will still guarantee this, but we do not make use of that property, so we should not require (and hence not guarantee) it.